api/billingd/service/service.go

```diff
@@ -1020,13 +1024,27 @@ func (s *Service) reportUsage() error {
 
 func (s *Service) reportCustomerUsage(ctx context.Context, cus *Customer) error {
 	for k, usage := range cus.DailyUsage {
+
+		summary := map[string]interface{}{
+			"grace_period_start":   s.analytics.FormatUnix(cus.GracePeriodStart),
+			"invoice_period_end":   s.analytics.FormatUnix(cus.InvoicePeriod.UnixEnd),
+			"invoice_period_start": s.analytics.FormatUnix(cus.InvoicePeriod.UnixStart),
+			"subscription_status":  cus.SubscriptionStatus,
+		}
 		if product, ok := s.products[k]; ok {
 			if err := s.reportUnits(product, usage, cus.ParentKey); err != nil {
 				return err
 			}
+
+			summary[product.Key+"_name"] = product.Name
+			summary[product.Key+"_usage"] = usage.Total
```

Comment:
Might be nice to keep this mapping exact like `_total`, and below, `_free_quota_size`

Reply:
oop, yep good call